guides/creating-a-custom-form-field-control.md

```diff
@@ -23,7 +23,7 @@ class MyTel {
 @Component({
   selector: 'example-tel-input',
   template: `
-    <div [formGroup]="parts">
+    <div role="group" [formGroup]="parts">
```

Comment:
FYI: This is the recommended pattern for grouping multiple inputs: https://www.w3.org/TR/WCAG20-TECHS/ARIA17.html

Reply:
@crisbeto do we have this on `date-range-input`?